src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -0,0 +1,59 @@
+import type { BtcTransactionStatus, BtcTransactionUi } from '$btc/types/btc';
+import type { BtcAddress } from '$lib/types/address';
+import type { BitcoinOutput, BitcoinTransaction } from '$lib/types/blockchain';
+import type { TransactionType } from '$lib/types/transaction';
+import { nonNullish } from '@dfinity/utils';
+
+export const mapBtcTransaction = ({
```

Comment:
Should we have a test for this or we will have an E2E tests?

Asking because generally now we provide tests when we add utils.

Reply:
I will add unit tests 👍

Reply:
Tests have been added.